packages/forklift-console-plugin/src/modules/Providers/ProviderRow.tsx

```diff
@@ -60,49 +49,52 @@ const fromProviderState = (status: ProviderStatus): 'Error' | 'Ok' | 'Unknown' |
   }
 };
 
-const toNegativeState = (conditionValue: string): 'Error' | 'Ok' | 'Unknown' => {
-  switch (conditionValue) {
-    case 'True':
-      return 'Error';
-    case 'False':
-      return 'Ok';
-    default:
-      return 'Unknown';
-  }
+const CategoryToIconMap = {
```

Comment:
In general, category is not sufficient to assign an icon. Keep in mind that conditions in K8s have 3 values: `true`, `false`, unknown. If we have a negative condition i.e.  `URLNotValid` but it's value is `false` then we should not display an icon suggesting error state. Similarly if we have a positive condition i.e. `Validated` then value `false` means that we have error state.
However as-for-now this will work - current backend code is not assigning values to conditions that are falsy i.e. `URLNotValid` is not present if the URL is valid.  Note that this may change in the future to be more k8s compliant (conditions should be orthogonal) .